lambda/README.md

```diff
@@ -0,0 +1,47 @@
+Contains a CloudFormation template to create a role and policy required 
+for the function along with the function code. Currently the function 
+and API are setup manually until we integrate building python, lambda, 
+api gateway and cloudformation into our build pipeline.
+
+Steps to deploy manually.
+Create the lambda function
+Create the gateway
+
+
+Here is some automation around this process.
+    
+# Package the script
+zip package.zip ./return_account_info.py
+
+# Create a new function
+aws lambda create-function \
+    --function-name getAccountList \
+    --runtime python2.7 \
+    --role arn:aws:iam::865492507168:role/RoleAWSADFSListAcounts \
```

Comment:
I think, that you placed here (by mistake probably) a real account number ;) Just for future notice, consider use something like 123456789012 instead ;)